docs/attendances_and_absences.md

```diff
@@ -0,0 +1,148 @@
+  > - there is currently no programmatic or user-facing way to do delete Wonderschool attendances
+  > - error handling
+
+#### _User-facing UI Attendance & Absence Creation - `AttendanceBatchesController#create` & `ServiceDaysController#create`_
+
+- for each record in the batch:
+  - if the record contains an attendance
+    - if a `ServiceDay` exists for that day for that child
+      - add an `Attendance` w/ the `check_in`, `check_out` associated to that `ServiceDay`
+    - if a `ServiceDay` does not exist for that day for that child
+      - create a `ServiceDay` for that day for that child
+      - add an `Attendance` w/ the `check_in`, `check_out`, associated to the new `ServiceDay`
+  - if the record contains an absence
+    - if the child is scheduled for that day
+      - if a `ServiceDay` exists for that day for that child
+        - change the `ServiceDay` to an absence w/ the `absence_type`
+      - if a `ServiceDay` does not exist for that day for that child
+        - create a `ServiceDay` for that day for that child w/ the `absence_type`
+    - if the child is not scheduled for that day
+      - if a `ServiceDay` exists for that day for that child
+        - delete the `ServiceDay`
+      - if a `ServiceDay` does not exist for that day for that child
+        - do nothing
+
+  > Behavior currently not defined 
+  > - error handling
+
+### Daily Automatic Absence Creation
+
+> `Nebraska::AbsenceGenerator`
+
+- for all children in Nebraska, generates an absence if there's no current attendance AND if the child is scheduled that day
+
+### Attendance Edit
+
+#### _User-facing UI Attendance Edit - `AttendancesController#update`_
+
+- if the update changes an existing attendance
+  - update the attendance w/ the `check_in`, `check_out`
+- if the update adds an additional attendance
+  - if a `ServiceDay` exists for that day for that child
+    - add an `Attendance` w/ the `check_in`, `check_out` associated to that `ServiceDay`
+  - if a `ServiceDay` does not exist for that day for that child
+    - create a `ServiceDay` for that day for that child
+    - add an `Attendance` w/ the `check_in`, `check_out`, associated to the new `ServiceDay`
+- if the update changes the ServiceDay to an absence
+  - if the child is scheduled for that day
+    - if a `ServiceDay` exists for that day for that child
+      - change the `ServiceDay` to an absence w/ the `absence_type`
+    - if a `ServiceDay` does not exist for that day for that child
+      - create a `ServiceDay` for that day for that child w/ the `absence_type`
+  - if the child is not scheduled for that day
+    - if a `ServiceDay` exists for that day for that child
+      - delete the `ServiceDay`
+    - if a `ServiceDay` does not exist for that day for that child
+      - do nothing
+
+  > Behavior currently not defined 
+  > - error handling
+
+### Attendance Deletion
+
+#### _User-facing UI Attendance Delete - `AttendancesController#destroy`_
+
+- delete the `Attendance` record
```

Comment:
@rebeccakarasiktw write a ticket to allow users to delete attendances in edit attendance

Reply:
Ticket written: #2691